### DISCUSSION TITLE
Does congnifly support latest iNav upstream?

### DISCUSSION THREAD

**Discussion by lida2003** [thecognifly/cognifly-python#2] [Q&A] [answered]
I notice [DRONE_SETUP](https://github.com/thecognifly/cognifly-python/blob/main/readme/DRONE_SETUP.md) uses quite old inav configurator (2.3.2) 

Does congnifly support latest code, such as iNav 5.x, [INAV 5.1 "Ballistic Buzzard", more specific](https://github.com/iNavFlight/inav/releases/tag/5.1.0)

Or should I use inav code in forked repo: https://github.com/thecognifly/inav

**Comment by lida2003** [comment, marked as answer]
It seems readme is quite out of date.

a) QA about cognifly-python's functions and features #1 , which suggests repo (yb/stable branch) works with INAV 6. 
b) [DRONE_SETUP](https://github.com/thecognifly/cognifly-python/blob/main/readme/DRONE_SETUP.md) is for 0.0.4 releases.

**Reply by yannbouteiller** [reply]
Yes it is because the `yb/stable` version is not officially out yet. It will be shortly after INAV6 is out as well.

At the moment, the version on PyPI and on the main branch is the old version that uses our old modded version of the 2.3.2 framework.

**Reply by lida2003** [reply]
Humm... That's good news, as dev is really close to the upstream. I'll use this branch as soon as google AI kit is received.

**Reply by lida2003** [reply]
as Oct 23, 2022  INAV 6 "Horizon Hawk" is released.
Sow which version do your suggest (work with cognify), lastested master code (self build) or INAV 6 "Horizon Hawk" ?

**Reply by yannbouteiller** [reply]
We use their master branch built with npm for the development of the `yb/stable` branch of cognifly-python

**Reply by yannbouteiller** [reply]
(But I expect their released INAV6 version will work if it is not a pre-release. I haven't tried yet.)

**Reply by lida2003** [reply]
OK. Thanks for the info.
